DancingLineECS/Assets/Scripts/ECS/Component/TransformRef.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace ECS.Component
```

Comment:
it is better to group by feature, not by type
e.g. instead of Component and System, you should have folders for Finish, Movement, etc.